silq/analysis/analysis.py

```diff
@@ -715,8 +717,81 @@ def analyse_EPR(empty_traces: np.ndarray,
             'mean_high_blip_duration': results_read['mean_high_blip_duration']}
 
 
-def analyse_flips(up_proportions_arrs: List[np.ndarray],
-                  threshold_up_proportion: Union[Sequence, float]):
+def determine_threshold_up_proportion(up_proportions_arr: np.ndarray,
```

Comment:
I've also given it some thought, and I think it makes sense to change the function name such that it reflects that this is only when a single state is measured. Because there's also another function (in another PR) that determines the threshold up proportion when multiple arrays are provided.

Maybe something like `determine_threshold_up_proportion_single_state`? If you have another suggestion let me know

Reply:
corrected.